tests/unit/test-api-search.js

```diff
@@ -7,43 +7,26 @@ function cloneMutatedItem() {
   return { ...item, links: item.links.slice(0) }
 }
 
-/*test('search es error', async (t) => {
-  const error = sinon.spy()
-  const proxyApi = proxquire('../libs/api', {
-    console: {
-      error
-    }
-  })
-  const errorMessage = 'errorMessage'
-  const search = sinon.stub().throws(new Error(errorMessage))
-  const backend = { search }
-  const response = await proxyApi.API('/', undefined, backend, 'endpoint')
-  t.is(error.firstCall.args[0].message, errorMessage,
-    'Logs Elasticsearch error via Winston transport')
-  t.is(response.message, errorMessage)
-  t.is(response.code, 500)
-})*/
-
-test('root /', async (t) => {
+test.skip('root /', async (t) => {
```

Comment:
were you planning to get these skipped tests working again?

Reply:
Yeah, but that's going to be part of #183